Editor/Credentials/Strings.cs

```diff
@@ -136,5 +136,21 @@ public static class Strings
         public const string TabAnywhere = "TabAnywhere";
         public const string TabEC2 = "TabEC2";
         public const string TabHelp = "TabHelp";
+        
```

Comment:
Could you please add in a screenshot for this page?

Reply:
Screen shot below